cmis/views/cmis_backend.xml

```diff
@@ -1,75 +1,74 @@
+                            <field name="location" placeholder="e.g. http://localhost:8081/alfresco/s/cmis" colspan="4" />
+                            <field name="username" colspan="2" />
+                            <field name="password" colspan="2" password="True" />
                         </group>
-                        <group name="directory_conf" string="Directory Configuration">
-                            <group colspan="4" col="4">
-                                <newline />
-                                <field name="initial_directory_write" />
-                                <button name="check_directory_of_write" type="object" class="oe_highlight" string="Test Directory" />
-                            </group>
+                    </group>
+                    <group name="directory_conf" string="Directory Configuration">
+                        <group colspan="4" col="4">
+                            <newline />
+                            <field name="initial_directory_write" />
+                            <button name="check_directory_of_write" type="object" class="oe_highlight" string="Test Directory" />
                         </group>
-                    </sheet>
-                </form>
-            </field>
-        </record>
+                    </group>
+                </sheet>
+            </form>
+        </field>
+    </record>
 
-        <record model="ir.ui.view" id="cmis_backend_search_view">
-            <field name="name">cmis.backend.search (in cmis)</field>
-            <field name="model">cmis.backend</field>
-            <field name="arch" type="xml">
-                <search string="CMIS Backend">
-                    <field name="name"/>
-                    <field name="location"/>
-                </search>
-            </field>
-        </record>
+    <record model="ir.ui.view" id="cmis_backend_search_view">
+        <field name="name">cmis.backend.search (in cmis)</field>
+        <field name="model">cmis.backend</field>
+        <field name="arch" type="xml">
+            <search string="CMIS Backend">
+                <field name="name"/>
+                <field name="location"/>
+            </search>
+        </field>
+    </record>
 
-        <record model="ir.ui.view" id="cmis_backend_tree_view">
-            <field name="name">cmis.backend.tree (in cmis)</field>
-            <field name="model">cmis.backend</field>
-            <field name="arch" type="xml">
-                <tree string="Claim Categories">
-                    <field name="name" widget="handle" />
-                    <field name="location"/>
-                </tree>
-            </field>
-        </record>
+    <record model="ir.ui.view" id="cmis_backend_tree_view">
+        <field name="name">cmis.backend.tree (in cmis)</field>
+        <field name="model">cmis.backend</field>
+        <field name="arch" type="xml">
+            <tree string="Claim Categories">
```

Comment:
@max3903 For sure I'm the author of this deprecated 'string' attribute... Nevertheless can you remove-it since it's deprecated and not related to cmis...